assets/js/header.js

```diff
@@ -63,3 +63,9 @@ function getCookie(f) {
     }
     return ''
 }
+
+
+var meta_nwds_ver = document.createElement('meta');
+meta_nwds_ver.name = 'ncbi_nwds_ver';
+meta_nwds_ver.content = nwds_version;  
+document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);
```

Comment:
They are meant to be included in the jsevent=render ping, which does not happen until the document is "ready" (I'm not 100% sure what that means).  Would it help to move this code to _footer.html?  That would come before any of the js files get included.  
However, I did find in some preliminary tests that the tag was logged as expected ... I can look again with this new version.